docs/src/pages/WebCheckboxPage/WebCheckboxPage.js

```diff
@@ -141,6 +141,31 @@ const components = [
       </form>,
     ],
   },
+  {
+    id: 'white',
+    title: 'Light',
+    dark: true,
+    examples: [
```

Comment:
Can you add a blurb prop to this too, just to describe what it does as I think it's not the most obvious (that it only changes the label, not the actual checkbox).

```suggestion
    blurb: 'You can change the label to white for displaying on dark backgrounds.',
    dark: true,
    examples: [
```

(Same comment for radio too).

Reply:
Sounds good!